src/simularium/types.ts

```diff
@@ -30,6 +30,9 @@ export interface CameraSpec {
     fovDegrees: number;
 }
 
+export type CameraSpecWithType = CameraSpec &
```

Comment:
can we just modify CameraSpec?

Reply:
The regular `CameraSpec` type is an element of `TrajectoryFileInfoV2`, which is read in from JSON. (The `VisGeometry.cameraDefault` property is read in from that field, and has that type.) Existing metadata ought to still get handled fine - anything that checked `orthographic` would get `undefined` and continue treating the spec as perspective - but I didn't want to touch anything at the file interface, and I'm not sure anyways whether it makes sense to provide the option to set the default camera as orthographic in the metadata.

Reply:
I would rather have the CameraSpec at the implementation level be the "complete" one, and change the one in TrajectoryFileInfoV2 to be PerspectiveCameraSpec.  That feels a little cleaner to me, especially since the TrajectoryFileInfo will probably be improved later on.  What do you think of that?  I feel like it would keep visGeometry a little cleaner and there could be a one-time-only conversion from PerspectiveCameraSpec to CameraSpec when `cameraDefault` is being set up.

Reply:
Makes sense to me